tests/serving/test_tracking.py

```diff
@@ -187,3 +188,20 @@ def test_tracking_datastore_profile(project: mlrun.MlrunProject) -> None:
     assert event["effective_sample_count"] == 2
     assert np.array_equal(event["request"]["inputs"], np.array([[0, -0.1], [0.4, 0]]))
     assert np.array_equal(event["resp"]["outputs"], np.array([0.0, 0.4 * 7]))
+
+
+def test_handle_endpoints_type_evaluate(rundb_mock):
+    project = "test-endpoints-handler"
+    _handle_endpoints_type_evaluate = (
+        ModelMonitoringApplicationBase._handle_endpoints_type_evaluate
+    )
+    parsed_endpoints = []
+    for endpoints in ["model-ep-1", ["model-ep-1"], [("model-ep-1", "model-ep-1-uid")]]:
+        parsed_endpoints.append(_handle_endpoints_type_evaluate(project, endpoints))
+
+    assert all(
+        list(
+            endpoint == [("model-ep-1", "model-ep-1-uid")]
+            for endpoint in parsed_endpoints
+        )
+    )
```

Comment:
Test parametrization will be more suitable here. It will allow detecting single output regressions